src/Uno.Toolkit.UI/Controls/ZoomContentControl/ZoomContentControl.xaml

```diff
@@ -0,0 +1,86 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:utu="using:Uno.Toolkit.UI">
+
+	<Style x:Key="DefaultZoomContentControlStyle"
+		   TargetType="utu:ZoomContentControl">
+		<Setter Property="Background"
+				Value="Transparent" />
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="utu:ZoomContentControl">
+					<Grid x:Name="PART_RootGrid"
+						  Background="{TemplateBinding Background}"
+						  BorderThickness="{TemplateBinding BorderThickness}"
+						  BorderBrush="{TemplateBinding BorderBrush}">
+						<Grid.RowDefinitions>
+							<RowDefinition Height="*" />
+							<RowDefinition Height="Auto" />
+						</Grid.RowDefinitions>
+						<Grid.ColumnDefinitions>
+							<ColumnDefinition Width="*" />
+							<ColumnDefinition Width="Auto" />
+						</Grid.ColumnDefinitions>
+						<Canvas x:Name="PART_Canvas">
+							<ContentPresenter x:Name="PART_Presenter"
+											  Grid.Row="0"
+											  Grid.Column="0"
+											  Margin="0"
+											  Padding="0"
+											  HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
+											  VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+											  Content="{TemplateBinding Content}"
+											  ContentTemplate="{TemplateBinding ContentTemplate}"
+											  ContentTemplateSelector="{TemplateBinding ContentTemplateSelector}">
+								<ContentPresenter.RenderTransform>
+									<TransformGroup>
+										<ScaleTransform CenterX="{TemplateBinding HorizontalZoomCenter}"
+														CenterY="{TemplateBinding VerticalZoomCenter}"
+														ScaleX="{TemplateBinding ZoomLevel}"
+														ScaleY="{TemplateBinding ZoomLevel}" />
+										<TranslateTransform X="{TemplateBinding HorizontalOffset}"
+															Y="{TemplateBinding VerticalOffset}" />
+									</TransformGroup>
+								</ContentPresenter.RenderTransform>
+							</ContentPresenter>
+						</Canvas>
+						<!-- Vertical ScrollBar -->
+						<ScrollBar x:Name="PART_ScrollV"
+								   Grid.Row="0"
+								   Grid.Column="1"
+								   HorizontalAlignment="Right"
+								   VerticalAlignment="Stretch"
+								   IndicatorMode="MouseIndicator"
+								   LargeChange="10"
+								   Maximum="{TemplateBinding VerticalMaxScroll}"
+								   Minimum="{TemplateBinding VerticalMinScroll}"
+								   Orientation="Vertical"
+								   SmallChange="1"
+								   Visibility="{TemplateBinding IsVerticalScrollBarVisible}"
+								   ViewportSize="{TemplateBinding ViewportHeight}"
+								   Value="{TemplateBinding VerticalScrollValue}" />
```

Comment:
VerticalScrollValue no longer exists, I think? So this must be breaking something.

And, as the PR is currently green, this means we are missing a test